tests/test_helpers/commands_helpers.ts

```diff
@@ -22,6 +26,12 @@ import { ScorecardChartDefinition } from "../../src/types/chart/scorecard_chart"
 import { SelectionDirection, SelectionStep } from "../../src/types/selection";
 import { target } from "./helpers";
 
+const size = { width: DEFAULT_FIGURE_WIDTH, height: DEFAULT_FIGURE_HEIGHT };
+const position = {
+  x: 0,
+  y: 0,
+};
```

Comment:
Try to use more specific variables names, something like `defaultFigureSize/position`. Otherwise its a bit confusing, even more so since this file contains lots of different stuff.